metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableBackend.java

```diff
@@ -472,8 +472,12 @@ public Statistics getStatistics() {
             fetches.add(readRows.directTransform(result -> {
                 for (final FlatRow row : result) {
                     watcher.readData(row.getCells().size());
-                    List<Metric> metrics = Lists.transform(row.getCells(), transform);
+                    final List<Metric> metrics = Lists.transform(row.getCells(), transform);
                     metricsConsumer.accept(MetricCollection.build(type, metrics));
+
+                    if (metrics.size() > 1) {
+                        watcher.reportRowDensity(getDensity(metrics));
```

Comment:
Two comments;

* It sucks that `metrics.size() == 0` is not well defined. That being said, I don't think we should just ignore it. It could be misrepresenting the data.
* Shouldn't this be done in `LocalMetricManager`? There's not specific dependency to BigtableBackend AFAICT.